opengrok-indexer/src/main/java/org/opengrok/indexer/web/PageConfig.java

```diff
@@ -763,13 +763,14 @@ public Annotation getAnnotation() {
      * current path is the "/" directory, otherwise set to an empty string.
      */
     public String[] getSearchOnlyIn() {
+        String path = getPath();
```

Comment:
How about revising line 768 rather than adding a local variable `path` hiding the field of same name

Reply:
oh, that's an omission. fixed.